packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -236,22 +250,32 @@ export class RedirectsMiddleware extends MiddlewareBase {
            * it returns `undefined`. The `matchedQueryString` is later used to indicate whether the query
            * string contributed to a successful redirect match.
            */
-          const matchedQueryString = this.isPermutedQueryMatch({
-            pathname: targetURL,
-            queryString: targetQS,
-            pattern: redirect.pattern,
-            locale,
-          });
+          if (req.headers.get('cdn-loop') === NAME_NETLIFY) {
```

Comment:
Is there a better way to safely check for netlify? Env variables?
Another question - do we really need to do this check, since :261 line does almost the same what `getPermutedQueryMatch` does. Does it really optimize something? since the same reg exp is executed